metrics-core/src/macros.rs

```diff
@@ -214,6 +219,9 @@ macro_rules! register_gauge {
     }
 }
 
+// TODO: do we need a specific macro to register other summary types? Would it
+// be one per type?
+
```

Comment:
I'm confused by this comment. The macros for the other summary types are right above this comment so I'm not sure what you're trying to say here.

Reply:
you're absolutely right. =)

Reply:
I thought we had been discussing histograms as just one type of summary structure. Do we need one specific to other approximate quantile algorithms? Are there other cases where we want to `register_summary` instead of `register_histogram`?

Reply:
Ah ok, this should just be renamed to `register_summary!` then.